_posts/2020-07-30-axon-kompleksowe-testowanie-aplikacji.md

```diff
@@ -0,0 +1,386 @@
+---
+layout:    post
+title:     "Axon - Kompleksowe testowanie aplikacji"
+date:      2020-07-30 08:00:00 +0100
+published: true
+author:    mkociszewski
+tags:
+    - java
+    - microservices
+    - axon
+    - event-sourcing
+    - spring-boot
+    - tests
+---
+
+Powszechnie wiadomo, że kod dobrze pokryty testami jest dużo bardziej podatny na rozwój - wszak nie musimy obawiać się, że nasza zmiana coś zepsuje, a my się o tym nie dowiemy.
+[W poprzednim wpisie]({% post_url 2020-06-08-microservices-on-axon %}) opisałem swoje zmagania z migracją monolitu do mikroserwisów na Axonie, nie wspomniałem tam jednak ani słowa o testach.
+Ten artykuł ma nadrobić zaległości w tej kwestii. Przedstawię dziś parę elementów składających się na kompleksowo przetestowaną aplikację opartą o Axona.
```

Comment:
mi 'zaległości' źle brzmią, czy to nie jest tak że to jest planowo druga część wpisu? :)

Reply:
Done